desktop-widgets/btdeviceselectiondialog.h

```diff
@@ -87,6 +91,9 @@ private slots:
 
 	void updateLocalDeviceInformation();
 	void initializeDeviceDiscoveryAgent();
+	void initializeDeviceList();
+
```

Comment:
you can remove this empty line.
normally we list member functions without empty lines separating them, unless for example we have groups of members that need to be separated.